.github/workflows/push.yml

```diff
@@ -0,0 +1,48 @@
+name: Check build
+
+on:
+  push:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  builddebug:
+    name: Debug
+    uses: klepsydra-technologies/kpe-ghworkflows/.github/workflows/build_project.yml@v2.0.6
+    with:
+      jobname: Build
+      unittests: true
+      cmakeopts: >
+        -DCMAKE_BUILD_TYPE=Debug -DKPSR_WITH_DOXYGEN=true
+        -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_CODE_METRICS=true
+    secrets: inherit
+
+  buildrelease:
+    needs: builddebug
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Vanilla
+            zmq: "false"
+            board: amd64
+          - name: ZMQ
+            zmq: "true"
+            board: amd64
+          - name: Vanilla aarch64
+            zmq: "false"
+            board: aarch64
```

Comment:
ok with aarch64, rpi4, odroid-xu4, and lx2160 boards for CE?

Reply:
This file should be similar to that of kpsr-core. So we don't use reusable workflows there, and only use the default (free) runners provided by github.

Reply:
Pretty sure the file has errors.